src/x11/tray_manager.cpp

```diff
@@ -67,25 +67,30 @@ tray_manager::~tray_manager() {
   deactivate();
 }
 
-void tray_manager::setup() {
+void tray_manager::setup(const string& tray_module_name) {
   const config& conf = config::make();
   auto bs = conf.section();
-  string position;
+  string position = "none";
 
   try {
     position = conf.get(bs, "tray-position");
   } catch (const key_error& err) {
-    return m_log.info("Disabling tray manager (reason: missing `tray-position`)");
   }
 
-  if (position == "left") {
-    m_opts.align = alignment::LEFT;
+  if (!position.empty() && !tray_module_name.empty()) {
```

Comment:
This will trigger whenever a tray module exists since `tray-position` will almost never be empty. I think it should check whether position is empty and that is it not `none`.